docs/technical-docs/zotero.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+title: Zotero
+parent: Technical Documentation
+nav_order: 5
+---
+
+# Zotero Tutorial
+
+An overview of what Zotero does, how to get started, and how to add to it.
+
+
+
+#### TABLE OF CONTENTS
+1. [What is Zotero?](https://ndclab.github.io/wiki/docs/zotero.html#what-is-zotero?)
+2. [Set-Up](https://ndclab.github.io/wiki/docs/zotero.html#set-up)
+3. [Saving Papers](https://ndclab.github.io/wiki/docs/zotero.html#saving-papers)
+4. [Folder Structure](https://ndclab.github.io/wiki/docs/zotero.html#folder-structure)
+5. [Creating and Navigating Tags](https://ndclab.github.io/wiki/docs/zotero.html#creating-and-navigating-tags)
+6. [Quick Tips](https://ndclab.github.io/wiki/docs/zotero.html#quick-tips)
+
+
+
+## What is Zotero?
+
+Zotero is a reference manager that allows the lab to have a shared collection of papers and makes it easy for users to add properly formatted in-text citations and bibliographies. 
+
+
+
+## Set-Up
+
+### Create an account
```

Comment:
Suggestion to help people identify all the steps: maybe number them?  Also, I think it might be more readable if we drop the header size by one:
#### 1. Create an account
#### 2. Get added...

Reply:
I was thinking the same thing about header size! Will integrate that and numbering them.